TimezoneConverter.xcodeproj/project.pbxproj

```diff
@@ -471,6 +483,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = macOS/macOS.entitlements;
+				CODE_SIGN_IDENTITY = "-";
```

Comment:
why do we have to change the codesign identity?

Reply:
Ugh I should've checked, sorry - I removed code sign identity from Build Settings. It should be gone when I push the rest of the changes

![image](https://user-images.githubusercontent.com/16971201/108578852-0eff1580-72d9-11eb-91f1-39b692825b22.png)